### DISCUSSION TITLE
Unable to access internet through IPv6 adresses on linux.

### DISCUSSION THREAD

**Discussion by MadProbe** [bol-van/zapret#192] [Q&A]
The problem is that I cannot get any http/s request done (infinite wait) on any opv6 address while requests upon ipv4 addresses work fine and I have my problem with connectivity solved just by running `sudo echo 1 >> /proc/sys/net/ipv6/conf/all/disable_ipv6` thus disabling ipv6 in my system completely.
Also pings to raw ipv6 addresses work fine as far as I remember, without disabling ipv6 obviously.
Could someone help with such an issue.
As I have previously stated in the title I am running this on Linux Mint 21 with tpws.
If you need more configuration files, ask me to show you them, and please give me paths or commands to pull out configs in this case.

**Comment by bol-van** [comment]
Did you run blockcheck on ipv6 hosts ?
If your strategy is wrong it can result in "hangs"
